app/Http/Requests/CreateProductRequest.php

```diff
@@ -25,6 +26,7 @@ public function rules(): array
             //
             'name'=>'required',
             'category'=>'required',
+            'price'=>'required',
             'description'=>'required',
             'date'=>'required',
```

Comment:
Let's make it best practice to add the datatype for stricter validation. For example...
- name = required, string
- price = required, numeric

As it stands, if a Unit Test were to try to pass an array to the name, it will cause an error in the backend, so let's make it a habit early on.